Documentation/tls/tls-certificates.md

```diff
@@ -0,0 +1,49 @@
+# Transport Layer Security (TLS) Certificates
+
+Tectonic secures all communications between cluster components. At install time, certificates can be provided in the following ways, from most automated to most customized:
+
+1. Installer generates a new CA, intermediate CAs and all certificates (default)
+2. Installer uses your provided CA (or intermediate) to generate all certificates
+3. Certificates are generated out-of-band (guides below) and provided to the Installer
+
+Read about the [TLS topology][tls-topology] for more technical information about the intermediate CAs used between components.
+
+## Components and Certificate Sources
+
+| Type of Traffic | Source of Certificate |
+|:----------|:----------------------|
+| API communication (external and internal) | Generated (default) </br> [User-provided][tls-identity] |
+| API to etcd cluster | Generated (default) </br> [User-provided][tls-etcd] |
+| Kubelet to anything | Generated (default) </br> [User-provided][tls-kube] |
+| Using the Tectonic Console and Ingress | Generated (default) </br> [User-provided][tls-ingress] |
+| Pod to Pod communication | Optionally secured by the application deployer |
+| Ingress to Pod communication | Optionally secured by the application deployer |
+
+### Default certificate generation
+
+The default behavior is to generate a new CA and use this to sign all certificates used in the cluster. This CA and the certificates will be stored on the machine where Tectonic Installer runs for your safe keeping.
+
+Users accessing the Tectonic Console may see browser warnings about untrusted self-signed certificates.
```

Comment:
To provide a different certificate for the Tectonic Console (link to tls-ingress) Follow these instructions.